sechub-api-java/src/main/java/com/mercedesbenz/sechub/api/User.java

```diff
@@ -0,0 +1,22 @@
+package com.mercedesbenz.sechub.api;
+
+public class User {
```

Comment:
You could use a record for this: https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/lang/Record.html

Reply:
Currently it would work with records - but in future we could have additional methods at this class.
Also we would have no `getUserId()` but instead `userId()` - because records do handle this different. Also equals should be implemented in way, that a set cannot contain the same use r with the same user id multiple times. Records would treat same user with different email addresses as different objects.

I will keep the class approach here.